src/main/config/simplified_tuning.c

```diff
@@ -77,61 +75,59 @@ static void calculateNewDTermFilterValues(pidProfile_t *pidProfile)
     if (pidProfile->dterm_lpf2_static_hz) {
         pidProfile->dterm_lpf2_static_hz = constrain(DTERM_LPF2_HZ_DEFAULT * pidProfile->simplified_dterm_filter_multiplier / 100, 0, LPF_MAX_HZ);
     }
-
-    if (!pidProfile->dterm_lpf1_type) {
-        pidProfile->dterm_lpf1_type = FILTER_PT1;
-    }
-
-    if (!pidProfile->dterm_lpf2_type) {
-        pidProfile->dterm_lpf2_type = FILTER_PT1;
-    }
 }
 
-static void calculateNewGyroFilterValues()
+static void calculateNewGyroFilterValues(gyroConfig_t *gyroConfig)
 {
-    if (gyroConfigMutable()->gyro_lpf1_dyn_min_hz) {
-        gyroConfigMutable()->gyro_lpf1_dyn_min_hz = constrain(GYRO_LPF1_DYN_MIN_HZ_DEFAULT * gyroConfig()->simplified_gyro_filter_multiplier / 100, 0, DYN_LPF_MAX_HZ);
-        gyroConfigMutable()->gyro_lpf1_dyn_max_hz = constrain(GYRO_LPF1_DYN_MAX_HZ_DEFAULT * gyroConfig()->simplified_gyro_filter_multiplier / 100, 0, DYN_LPF_MAX_HZ);
-    }
-
-    if (gyroConfigMutable()->gyro_lpf1_static_hz) {
-        gyroConfigMutable()->gyro_lpf1_static_hz = constrain(GYRO_LPF1_DYN_MIN_HZ_DEFAULT * gyroConfig()->simplified_gyro_filter_multiplier / 100, 0, DYN_LPF_MAX_HZ);
-    }
-
-    if (gyroConfigMutable()->gyro_lpf2_static_hz) {
-        gyroConfigMutable()->gyro_lpf2_static_hz = constrain(GYRO_LPF2_HZ_DEFAULT * gyroConfig()->simplified_gyro_filter_multiplier / 100, 0, LPF_MAX_HZ);
+    if (gyroConfig->gyro_lpf1_dyn_min_hz) {
+        gyroConfig->gyro_lpf1_dyn_min_hz = constrain(GYRO_LPF1_DYN_MIN_HZ_DEFAULT * gyroConfig->simplified_gyro_filter_multiplier / 100, 0, DYN_LPF_MAX_HZ);
```

Comment:
This line is too wide for the reviewing view. It would be good to try to keep to 80 chars per line. Just an observation.

Reply:
Hey that is not mine LOL. I will fix it

Reply:
Nooooooo